content/600-about/200-prisma-docs/20-style-guide/03-spelling-punctuation-formatting.mdx

```diff
@@ -46,15 +46,19 @@ Although you might be tempted to use abbreviations like "Postgres" (instead of t
 - MongoDB (instead of Mongo)
 - GitHub (instead of Github)
 - For JSON, see [Special case: JSON](#special-case-json)
+- VS Code (this is an official Microsoft short form, so is acceptable. Be sure to include the space. Do not use the long form Visual Studio Code)
 
 If you're not sure about the spelling of a proper noun, check its official web site.
 
 ### Titles and headings
 
-- Use sentence case for titles and headings: only the initial word is uppercase (exception: [capitalize proper nouns](#capitalize-and-spell-out-proper-nouns))
-- [Avoid gerunds](/about/prisma-docs/style-guide/word-choice#avoid-gerunds-ing-verb-forms) ("Configure the database", not "Configuring the database")
-- Do not use punctuation at the end of a title or heading unless a question mark is required
-- Use `<inlinecode>code</inlinecode>` for code in headings - this is required by our navigation elements
+- Use sentence case for titles and headings: only the initial word is uppercase (exception: [capitalize proper nouns](#capitalize-and-spell-out-proper-nouns)).
+- [Avoid gerunds](/about/prisma-docs/style-guide/word-choice#avoid-gerunds-ing-verb-forms) ("Configure the database", not "Configuring the database").
+- Do not use punctuation at the end of a title or heading. Exception: question marks are OK.
+- Use `<inlinecode>code</inlinecode>` for code in headings - this is required by our navigation elements.
+- Do not place a heading immediately at the top of a page, below the page title and before the first body text. It's implicit that the first paragraph on a page is an overview or introduction. There's no need to have a heading stating this.
```

Comment:
The heading leads to it being present in the right hand sidebar. Sometimes that is more important than the visual display I would say.

Reply:
@janpio  Is your concern that someone who has scrolled down the page can't jump back up to the intro because it doesn't have a heading?

Reply:
Yes. Often that content is the most important piece of the whole page, what describes a concept or feature. The right hand sidebar should be a table of contents.

Reply:
I came up with a solution to this that would please everyone, but it would require a bit of webdev work. In the circumstances, I'll remove this rule.

Reply:
I came up with a solution for this, but it requires some webdev work. Under the circumstances, I'll remove this addition to the style guide.